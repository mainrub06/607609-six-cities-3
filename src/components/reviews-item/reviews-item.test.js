import React from "react";
import renderer from "react-test-renderer";
import ReviewsItem from "./reviews-item.jsx";
import REVIEWS from "../../mock/reviews";

it(`Render ReviewsItem component`, () => {
  const tree = renderer
    .create(
        <ReviewsItem element = {REVIEWS[0].reviewsArr[0]}/>
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
