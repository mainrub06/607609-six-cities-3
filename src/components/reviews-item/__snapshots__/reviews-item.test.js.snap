// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render ReviewsItem component 1`] = `
<li
  className="reviews__item"
>
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Avatar"
        className="reviews__avatar user__avatar"
        height="54"
        src="img/avatar-max.jpg"
        width="54"
      />
    </div>
    <span
      className="reviews__user-name"
    >
      Luis
    </span>
  </div>
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": 80,
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    >
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam lectus mauris, commodo eget malesuada id, gravida id mi.
    </p>
    <time
      className="reviews__time"
      dateTime="2019-04-24"
    >
      December 24, 2018
    </time>
  </div>
</li>
`;
